# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# Filename: target/mnemosyne/pkg_diff/sudo_xfsm.diff
# Copyright (C) 2002 - 2006 Alejandro Mery
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- ./sudoers.orig	2005-11-27 02:53:27.000000000 -0300
+++ ./sudoers	2005-11-27 02:58:52.000000000 -0300
@@ -29,0 +30 @@
+%users	ALL=(root) NOPASSWD: /opt/xfce4/libexec/xfsm-shutdown-helper
